### PR TITLE
Review IGs 202212.0

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description
The review of IGs in `_includes/pbc/all/install-features/202212.0`
TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _includes/pbc/all/install-features/202212.0/install-the-acl-feature.md [hunk 1]
```diff
@@ -273,26 +276,19 @@ class MerchantPortalApplicationDependencyProvider extends SprykerMerchantPortalA
...
-Make sure that `spy_acl_role`, `spy_acl_group`, and `spy_acl_user_has_group` tables contain default data.
-
-Make sure that you can edit user's ACL groups on User edit page in Back Office.
-
-Make sure that `AclEntityRule` is created in `spy_acl_entity_rule` when the `RoleTransfer` is saved and contains `AclEntityRules`.
-
-Make sure that `RolesTransfer` contains needed `AclEntityRules`.
-
-Ensure the user who is not supposed to have access to an entity or endpoint does not have it.
+Make sure the following:
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202212.0/install-the-acl-feature.md:285]:
```suggestion
Make sure the following works correctly:
```

File: _includes/pbc/all/install-features/202212.0/install-the-approval-process-feature.md [hunk 2]
```diff
@@ -508,22 +503,21 @@ class CheckoutPageDependencyProvider extends SprykerShopCheckoutPageDependencyPr
 
 {% info_block warningBox "Verification" %}
 
-Check that the customer with the sent approval request cannot open the address step on the cart page.
-
-Check that the customer with the sent approval request cannot open the payment step on the cart page.
+On the storefront, make sure the following:
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202212.0/install-the-approval-process-feature.md:506]: same

File: _includes/pbc/all/install-features/202212.0/install-the-availability-notification-feature.md [hunk 3]
```diff
@@ -289,31 +284,30 @@ class AvailabilityNotificationConfig extends SprykerAvailabilityNotificationConf
 
 {% info_block warningBox “Verification” %}
 
-
 We recommend setting `AVAILABILITY_NOTIFICATION_CHECK_PRODUCT_EXISTS` to true. 
-Make sure that you are not catching the mentioned above exception somewhere in your Pyz code but use check of `$availabilityNotificationSubscriptionResponseTransfer->getIsSuccess()`.
+Make sure that don't catch the previously mentioned exception somewhere in your Pyz code but use the check of `$availabilityNotificationSubscriptionResponseTransfer->getIsSuccess()`.
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202212.0/install-the-availability-notification-feature.md:288]: Double-check this sentence

File: _includes/pbc/all/install-features/202212.0/install-the-approval-process-feature.md (hunk 2 above)
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202212.0/install-the-approval-process-feature.md:506]: @andriitserkovnyi I've merged them. Don't think there's a need to keep them separately.
```suggestion
On the storefront, make sure that the customer with the sent approval request can't open the address and payment steps on the cart page.
```

File: _includes/pbc/all/install-features/202212.0/install-the-availability-notification-feature.md (hunk 3 above)
Comment (VadymSachenko) [reply _includes/pbc/all/install-features/202212.0/install-the-availability-notification-feature.md:288]: Rewritten
```suggestion
Make sure that the previously mentioned exception is not thrown somewhere in your Pyz code; however, use the check of `$availabilityNotificationSubscriptionResponseTransfer->getIsSuccess()`.
```

File: _includes/pbc/all/install-features/202212.0/install-the-approval-process-feature.md [hunk 4]
```diff
@@ -255,13 +252,11 @@ Go to the Back Office, **Maintenance** menu, and click **Sync permissions**.
...
 {% info_block warningBox "Verification" %}
 
-Check that the customer with the permission `RequestQuoteApprovalPermission` can request for approval.
-
-Check that the customer with the permission `ApproveQuotePermission` can approve the request.
-
-Check that the customer with the permission `PlaceOrderPermissionPlugin` can place an order from the quote with the approved request for approval.
-
-Check that when you reset the cart lock, all the approval process-related data is removed from the quote.
+Check the following:
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202212.0/install-the-approval-process-feature.md:255]:
```suggestion
Check that the following happens:
```

File: _includes/pbc/all/install-features/202212.0/install-the-approval-process-feature.md (hunk 2 above)
Comment (VadymSachenko) [reply _includes/pbc/all/install-features/202212.0/install-the-approval-process-feature.md:506]: @andriitserkovnyi I've merged them. Don't think there's a need to keep them separately.
```suggestion
On the storefront, make sure that the customer with the sent approval request can't open the address and payment steps on the cart page.
```

**Review by andriitserkovnyi — APPROVED** [review]

**Pull request merged**